openpype/hosts/houdini/plugins/create/create_unreal_staticmesh.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating Unreal Static Meshes.
+
+Unreal Static Meshes will be published as FBX.
+
+Filmbox by default expects an ObjNode
+however, we set the sop node explictly
+to eleminate any confusion.
+
+This will make Filmbox to ignore any object transformations!
+
+get_obj_output selects
+the output sop with mimimum idx
+or the node with render flag isntead.
```

Comment:
```suggestion
get_obj_output selects
the output sop with mimimum idx
or the node with render flag instead.
```

But it's probably best to just remove it:
```suggestion
```

Reply:
please bare with me, I'll remove a lot of comments in upcoming commits